packages/office-ui-fabric-react/src/components/Overlay/Overlay.styles.ts

```diff
@@ -0,0 +1,52 @@
+import { IOverlayStyleProps, IOverlayStyles } from './Overlay.types';
+import {
+  IStyle,
+  ITheme,
+  HighContrastSelector,
+} from '../../Styling';
+
+export const getStyles = (
+  props: IOverlayStyleProps
+): IOverlayStyles => {
+  const {
+    className,
+    theme,
+    isNone,
+    isDarkThemed,
+  } = props;
+
+  const { palette, semanticColors } = theme;
+
+  return ({
+    root: [
+      'ms-Overlay',
+      {
+        backgroundColor: palette.whiteTranslucent40,
+        top: 0,
+        right: 0,
+        bottom: 0,
+        left: 0,
+        position: 'absolute',
+
+        selectors: {
+          [HighContrastSelector]: {
+            border: '1px solid WindowText',
+          }
+        }
+      },
+
+      isNone && {
+        visibility: 'hidden',
+      },
+
+      isDarkThemed && [
```

Comment:
Could we rename this to `isDark`? That would match the CSS class below, and remove any potential confusion around this prop and theming.

Reply:
I'll have to deprecate it but sure.

Reply:
Even if we don't change the component's prop, this style prop could be changed for now right?

Reply:
Ah, yeah.